lib/Doctrine/Persistence/Mapping/Driver/FileLocator.php

```diff
@@ -26,28 +24,22 @@ public function findMappingFile($className);
      *
      * @return string[]
      */
-    public function getAllClassNames($globalBasename);
+    public function getAllClassNames(?string $globalBasename = null) : array;
```

Comment:
Why making it nullable and having default value? It breaks contract

Reply:
We had some instances in the tests that passed null. Let me see what I can do make this better.